hibernate-core/src/test/java/org/hibernate/orm/test/metamodel/mapping/collections/MapOperationTests.java

```diff
@@ -47,8 +47,11 @@ public void createData(SessionFactoryScope scope) {
 					entityContainingMaps.addBasicByBasic( "someKey", "someValue" );
 					entityContainingMaps.addBasicByBasic( "anotherKey", "anotherValue" );
 
-					entityContainingMaps.addSortedBasicByBasic( "key1", "value1" );
 					entityContainingMaps.addSortedBasicByBasic( "key2", "value2" );
+					entityContainingMaps.addSortedBasicByBasic( "key1", "value1" );
```

Comment:
It doesn't make sense to insert testing data with the natural order. Ideally we are supposed to populate big number of data to ensure the order is not maintained not by accident.